pvnet_app/app.py

```diff
@@ -1,13 +1,4 @@
-"""App to run inference
-
-This app expects these evironmental variables to be available:
-    - DB_URL
-    - NWP_UKV_ZARR_PATH
-    - NWP_ECMWF_ZARR_PATH
-    - SATELLITE_ZARR_PATH
-    - RUN_EXTRA_MODELS
-    - USE_ADJUSTER
-    - SAVE_GSP_SUM
+"""App to run inference for PVNet models
```

Comment:
Could you add the relevant env vars bk in please

Reply:
The environmental vars are also in the docstring for the `app()` function. I thought it was messy to have them in two places in the one file. Plus these are now out of date. What do you think?

Reply:
if they are up to date in the doce string, then maybe thats ok. Are they up to date on the readme too?
Its probably worth adding them in on the readme, but that could be a seperate issue if you like

Reply:
Yeh they are up to date in the docstring. We've never had them (or any usage info) in the README, so I think if we want that it should be a separate issue